docs/release-notes/2.43.0.md

```diff
@@ -183,6 +183,8 @@ If you're looking to upgrade, you can easily [install the latest version of Perc
   
 - [PMM-13292](https://perconadev.atlassian.net/browse/PMM-13292) - Introduced an improved mechanism to resolve port conflicts when starting multiple PMM agents on the same machine, enhancing reliability and reducing manual intervention in complex monitoring setups.
 
+- [PMM-13141](https://perconadev.atlassian.net/browse/PMM-13141) - Introduced new MongoDB metrics for Feature Compatibility Version (FCV), providing insights into the MongoDB version compatibility of individual nodes, enhancing monitoring capabilities for multi-version MongoDB deployments.
```

Comment:
```suggestion
- [PMM-13141](https://perconadev.atlassian.net/browse/PMM-13141) - Introduced new MongoDB metrics for Feature Compatibility Version (FCV),  enhancing monitoring capabilities for multi-version deployments. 

Collected via the MongoDB exporter, this metric provides insights into individual node compatibility and supports upcoming PMM advisors. 

To use this feature, enable the FCV collector with the `--enable-all-collectors` flag when adding MongoDB services to PMM.
```

Reply:
it's actually enabled by default after a comment from @igroene

Reply:
thanks @BupycHuk , I've updated my  suggestion